PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,7 +25,7 @@ and the release note for this PR. For example:
   * Added a new solver for logarithmic equations.
 
 * functions
-  * Fixed a bug with log of integers.
+  * Fixed a bug with log of integers. Formerly, `log(-x)` incorrectly gave `-log(x)`.
```

Comment:
Let's also add something that says what happens now like

> Now `log(-x)` will remain as `log(-x)` unless `x` is an explicit number whose sign can be computed (for example `log(-3)` evaluates to `log(3) + I*pi`).

Reply:
Can we add other example instead of  `log`

Reply:
Why would a different example be better?

Reply:
Or we move ahead with that?
* solvers
  * Added a new solver for logarithmic equations.
* functions
  * Fixed a bug with log of integers. Formerly, `log(-x)` incorrectly gave `-log(x)`. Now log(-x) will remain as log(-x) unless x is an explicit number whose sign can be computed (for example log(-3) evaluates to log(3) + I*pi).

Reply:
We can find something that is simpler but the example release notes should demonstrate doing these things:
1. Describe what is changed and why in a way that makes sense for users.
2. Clearly state what the previous behaviour was.
3. Clearly state what the new behaviour is.
4. Give an explicit example of the behaviour that is referred to.

Not every release note needs all of those but the example release notes should show them for contributors to think about when writing a release note. I also think that "Added a new solver for logarithmic equations" is not good because it doesn't even say what end user function is affected by the change but there an example would be fine e.g. "For example `solve(log(a) - log(2*x), x)` now returns `[a/2]`".

Reply:
* physics.units
  * Corrected a numerical error in the conversion of volt and statvolt
* functions
  * Formerly, it gaves 1 volt = 299.792458 statvolts, Now it gives 1 volt =  0.00333564095198152*statvolt.

Reply:
* physics.units
    * Corrected a numerical error in the conversion between `volt` and `statvolt`. Previously, 1 volt was converted to 299.792458 statvolts. Now the correct formula is used and gives 1 volt = 0.00333564095198152 statvolts.

Reply:
pushed a commit with that example.